### PR TITLE
Updates to add <kevm> wrapper cell to EVM configuration

### PR DESCRIPTION
**Pull request by ehildenb**
In reference to: https://github.com/kframework/evm-semantics/pull/335

### CODE REVIEW

**Comment by ehildenb** [comment]
@dwightguth the second commit is needed to get it to build on KEVM with configuration changes: https://github.com/kframework/evm-semantics/pull/335

Without this change, I get the following error when doing `make build-ocaml` on KEVM:

```
File "world.mli", line 33, characters 54-75:
Error: Unbound type constructor Msg_types.iele_config
Hint: Did you mean hello_config?
make: *** [Makefile:187: .build/plugin-ocaml/semantics.cmxa] Error 2
```

File: vm-c/vm.cpp [hunk 1]
```diff
@@ -203,7 +203,7 @@ CallResult run_transaction(CallContext ctx) {
   map withSched = hook_MAP_element(configvar("$SCHEDULE"), (block *)scheduleinj);
   map withMode = hook_MAP_update(&withSched, configvar("$MODE"), (block *)modeinj);
   map init = hook_MAP_update(&withMode, configvar("$PGM"), inj);
-  static uint32_t tag2 = getTagForSymbolName("LblinitGeneratedTopCell{}");
+  static uint32_t tag2 = getTagForSymbolName("LblinitKevmCell{}");
```
Comment (dwightguth) [inline vm-c/vm.cpp]: Please revert this line

Comment (ehildenb) [reply vm-c/vm.cpp]: Done.

**Comment by ehildenb** [comment]
@dwightguth confirmed that with that line reverted, still builds on KEVM.

**Pull request merged**